poll/poll_test.go

```diff
@@ -18,14 +18,14 @@ func (t *fakeT) Fatalf(format string, args ...interface{}) {
 	panic("exit wait on")
 }
 
-func (t *fakeT) Log(args ...interface{}) {}
+func (t *fakeT) Log(...interface{}) {}
 
-func (t *fakeT) Logf(format string, args ...interface{}) {}
+func (t *fakeT) Logf(string, ...interface{}) {}
```

Comment:
Maybe you coule use `any` instead of `interface{}`

Reply:
I kept `interface{}` for now, so that we didn't have to raise the go version in `go.mod`; https://github.com/gotestyourself/gotest.tools/blob/631f6c882e6778087ca9eb2074775835fdb4737f/go.mod#L3

(`any` would require go1.18 as minimum; https://go.dev/ref/spec#Go_1.18)

Reply:
I know 😬 but Go 1.18 is now not only old, but no longer supported

Reply:
Yup, more than aware; I also know that it's not uncommon for companies to maintain an (internal) LTS version of older Go versions. Not all of those will be visible publicly on GitHub. (e.g. I know Google for a long time required compatibility with go1.13 for some of their systems - they maintained their own version for that).